### PR TITLE
Add OCP templates

### LINKED ISSUE #6: "Run on Openshift": Create OCP template
**Is your feature request related to a problem? Please describe.**
I would like to have also an OpenShift template for deploying only windup-api.

**Describe the solution you'd like**
Create a yaml template.

**Describe alternatives you've considered**
-

**Additional context**
-

### LINKED ISSUE #9: "Run on Openshift": add the paragraph
**Is your feature request related to a problem? Please describe.**
Provide commands to deploy windup-api on Openshift.

**Describe the solution you'd like**
Provide the required commands for running on OCP.

**Describe alternatives you've considered**

**Additional context**

### LINKED ISSUE #13: "Try with Tackle": Provide OCP deployment instructions and assets
**Is your feature request related to a problem? Please describe.**

There is no provided way to deploy the "Try with Tackle" example in OCP. 

**Describe the solution you'd like**

Provide deployment models for both Minikube and OCP for the "Try with Tackle" section.

**Describe alternatives you've considered**
None

**Additional context**
None

### PR DESCRIPTION
**Pull request by carlosthe19916**
This PR includes:
- [x] OCP templates
- [x] instructions about how to use the OCP templates

resolves #6  ,resolves #9  ,resolves #13  
rendered version available https://github.com/carlosthe19916/windup-api/blob/ocp-templates/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,15 +140,102 @@ Now you can move to the [Usage Guide - API with Tackle](#api-with-tackle-2) to g
...
 
 ### API with sample page
 
-TBD  
+On your OCP instance, create a project `windup` executing
+```shell
+oc new-project windup
+```
+
+then upload our template to your project executing
```
Comment (mrizzi) [inline README.md:152]:
```suggestion
then upload API's template to your project executing
```

File: README.md [hunk 2]
```diff
@@ -140,15 +140,102 @@ Now you can move to the [Usage Guide - API with Tackle](#api-with-tackle-2) to g
...
+An early (i.e. proof of concept) integration between [Tackle](https://github.com/konveyor/tackle), the tools that support the modernization and migration of applications to Kubernetes from [Konveyor](https://www.konveyor.io/) community, and Windup API is available.
+
+A kubernetes manifest for deploying the integrated versions of Tackle and Windup API is provided.  
+
+On your OCP instance, create a project `windup` executing
+```shell
+oc new-project windup
+```
+
+then upload our template to your project executing
```
Comment (mrizzi) [inline README.md:198]:
```suggestion
then upload API's template to your project executing
```

File: README.md [hunk 3]
```diff
@@ -140,15 +140,102 @@ Now you can move to the [Usage Guide - API with Tackle](#api-with-tackle-2) to g
...
+```shell
+oc wait -n windup --for condition=Available deployment windup-api --timeout=-1s
+```
+
+As soon as the `windup-api` deployment will be available, the following message will be displayed:
+```shell
+deployment.apps/windup-api condition met
+```
+
+Now you can start testing the Windup API opening, in your browser, the Route url provided by OCP. To see the routes available in your project execute:
```
Comment (mrizzi) [inline README.md:174]:
```suggestion
Now you can start testing the Windup API opening, in your browser, the Route url provided by OCP.  
The route's URL is the value provided in the `HOST/PORT` column in the output of the command:
```

File: README.md [hunk 4]
```diff
@@ -140,15 +140,102 @@ Now you can move to the [Usage Guide - API with Tackle](#api-with-tackle-2) to g
...
+deployment.apps/pathfinder-postgres condition met
+deployment.apps/tackle-application-inventory condition met
+deployment.apps/tackle-controls condition met
+deployment.apps/tackle-pathfinder condition met
+deployment.apps/tackle-ui condition met
+deployment.apps/windup-api condition met
+deployment.apps/windup-executor condition met
+```
+
+Once deployed, the Tackle UI can be opened through the Route provided by OCP. To see the routes available execute
```
Comment (mrizzi) [inline README.md:229]:
```suggestion
Once deployed, the Tackle UI can be opened through the Route provided by OCP.  
The route's URL is the value provided in the `HOST/PORT` column in the output of the command:
```

**Review by mrizzi — COMMENTED** [review]
Some minor wording changes but the commands worked fine for me :+1:

**Comment by rromannissen** [comment]
LGTM

**Review by mrizzi — APPROVED** [review]
Thanks @carlosthe19916 :+1: 
@rromannissen found it great as well so I'm going to merge.

**Pull request merged**